src/components/ConnectToNerbordButton/ConnectToNerdbordButton.tsx

```diff
@@ -0,0 +1,41 @@
+﻿'use client'
+import { connectToNerdbord } from '@/actions/user/connectToNerdobord'
+import { Button } from '../Button/Button'
+import RotateIcon from '@/assets/icons/RotateIcon'
+import { ToastStatus, useToast } from '@/contexts/ToastContext'
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+
+export const ConnectToNerdbordButton = () => {
+  const { addToast } = useToast()
+  const [loading, setLoading] = useState(false)
+  const router = useRouter()
+
+  const handleConnectToNerdbord = async () => {
+    setLoading(true)
+    try {
```

Comment:
**nitpick:** Why not `useRunAsync`?

Reply:
I had a problem getting the error from the hook because the error was displayed in the console by the `withErrorHandlingAsync`.